### DISCUSSION TITLE
Vanity domains

### DISCUSSION THREAD

**Discussion by elreydetoda** [JupiterBroadcasting/jupiterbroadcasting.com#190] [Ideas]
So, @thechangelog's website has a concept of a "vanity domain" (don't know if that's what it's actually called but that's how I've heard it 😅). It's not the domain for the show, but it's a short domain that they can share over the air or on social media sites (IIRC I think they initially did it because sites like Twitter used to count your URL length against your character count).

This makes it easier for them to say (i.e.  instead of saying linuxunplugged.com) and easier for people to navigate to.

So for example one of their shows is located at: https://changelog.com/shipit

But the vanity domain for it is: https://shipit.show and if you do something like https://shipit.show/62 it'll take you to episode 62.

Same thing for this show: https://changelog.com/jsparty

Vanity domain: https://jsparty.fm/235

I think the .fm domains are probably more appropriate for JB, but honestly it'll be up to price and availability. I think it should try and be standardize though if we can.

Also, doing some quick checks myself it looks like (don't know your normal registrar) namecheap instead of domain.com because I looked up lup.fm (which is available) on domain.com and it was like ~$400+, but on namecheap it was ~$100 😅

**Comment by gerbrent** [comment]
related to #161 

also less of an issue and more of an idea/discussion, so converting as so.
